modules/dynamic-plugin-localization.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin/overview-dynamic-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="dynamic-plugin-api_{context}"]
+= Translating messages with react-i18next
+
+The link:https://github.com/openshift/console-plugin-template[plugin template] demonstrates how you can translate messages with link:https://www.i18next.com/[react-i18next]. The `v1alpha1` `apiVersion` of `ConsolePlugin` supports `console.openshift.io/use-i18n` annotation.
```

Comment:
@jhadvig Have we removed `v1alpha1`?

Reply:
Removed it since I dont think it is really necessary to add. I can add it back though if you think it is essential information to have.